.env.example

```
@@ -0,0 +1,21 @@
+
+# MongoDB Configuration
+MONGO_URI=mongodb://mongodb:27017/campus-marketplace
+
+ACCESS_TOKEN_SECRET=your-secure-access-token-secret-here
+REFRESH_TOKEN_SECRET=your-secure-refresh-token-secret-here
+
+# CORS Configuration
+CORS_ORIGIN=http://localhost:5173
+
+CLOUDINARY_CLOUD_NAME=your-cloudinary-cloud-name
+CLOUDINARY_API_KEY=your-cloudinary-api-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLOUDINARY_API_KEY key should go before the CLOUDINARY_CLOUD_NAME key

(UnorderedKey)